### PR TITLE
Feat: remove howto footer, add comment focus

### PR DESCRIPTION
**Pull request by AlfonsoGhislieri**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

Removes footer for howto's as suggested by @davehakkens  in #2774, also adds focus to the comments form when pressing leave a comment button for a better use experience

## Git Issues

Closes #

## Screenshots/Videos

_If useful, provide screenshot or capture to highlight main changes_

---

## What happens next?

Thanks for the contribution! We try to make sure all PRs are reviewed ahead of a monthly dev call (first Monday of the month, open to all!).

If the PR is working as intended it'll be merged and included in the next platform release, if not changes will be requested and re-reviewed once updated.

If you need more immediate feedback you can try reaching out on Discord in the [Community Platform `development` channel](https://discord.com/channels/586676777334865928/938781727017558018).

### CODE REVIEW

**Comment by davehakkens** [comment]
Looks good to me! 👍

File: src/pages/Howto/Content/Howto/Howto.tsx [hunk 1]
```diff
@@ -11,20 +10,12 @@ import type { RouteComponentProps } from 'react-router'
...
 import { HowToComments } from './HowToComments/HowToComments'
 import HowtoDescription from './HowtoDescription/HowtoDescription'
 import { isAllowedToEditContent } from 'src/utils/helpers'
 import { seoTagsUpdate } from 'src/utils/seo'
 import Step from './Step/Step'
-// TODO: Remove direct usage of Theme
-import { preciousPlasticTheme } from 'oa-themes'
-const theme = preciousPlasticTheme.styles
-import { Link } from 'react-router-dom'
-import WhiteBubble0 from 'src/assets/images/white-bubble_0.svg'
```
Comment (thisislawatts) [inline src/pages/Howto/Content/Howto/Howto.tsx:24]: **issue(blocking):** It looks like these images are not being used anywhere else. Can we remove them please 🙏🏻

**Review by thisislawatts — CHANGES_REQUESTED** [review]

File: src/pages/Howto/Content/Howto/Howto.tsx (hunk 1 above)
Comment (AlfonsoGhislieri) [reply src/pages/Howto/Content/Howto/Howto.tsx:24]: The images are being use by the [`MoreContainer` component](https://github.com/ONEARMY/community-platform/blob/master/packages/components/src/MoreContainer/MoreContainer.tsx) that is used inside the HowtoList.

Comment (thisislawatts) [reply src/pages/Howto/Content/Howto/Howto.tsx:24]: Apologies for the delay. These images are duplicated and also exist within the `packages/components` directory, so we can remove them from the root of project.

Comment (AlfonsoGhislieri) [reply src/pages/Howto/Content/Howto/Howto.tsx:24]: Ahh good catch @thisislawatts of course... Removed them in [4b553cb](https://github.com/ONEARMY/community-platform/pull/2779/commits/4b553cbd4b59b4fad465e74a699a122b10b89823)

**Review by thisislawatts — APPROVED** [review]

**Pull request merged**